views/js/qtiXmlRenderer/renderers/Container.js

```diff
@@ -79,4 +81,27 @@ define(['tpl!taoQtiItem/qtiXmlRenderer/tpl/container'], function(tpl){
             return data;
         }
     };
+
+    /**
+     * Merges specified sibling html tags into one
+     * @param {string} html
+     * @returns {string}
+     */
+    function mergeSiblings(html)
+    {
+        if (typeof html !== 'string' || html.length <= 0) {
+            return html;
+        }
+        const tagsToMerge = [
+            'strong',
+            'em',
+            'sub',
+            'sup'
+        ];
+        tagsToMerge.forEach(function(tag) {
+            let regex = new RegExp("<\\/"+tag+">(\\s*)<"+tag+">", 'gi');
```

Comment:
eslint issue
<img width="730" alt="image" src="https://user-images.githubusercontent.com/25976342/217246963-22049b84-a1cd-491d-b524-dcceffd708d2.png">
try this
```suggestion
            let regex = new RegExp(`<\\/${tag}>(\\s*)<${tag}>", 'gi');
```

Reply:
Changed :)